src/main/java/com/flansmod/client/model/RenderGun.java

```diff
@@ -263,6 +264,32 @@ private void renderWeaponSwitchMovement(GunAnimations animations) {
 		GL11.glRotatef(startAngles.z + (endAngles.z - startAngles.z) * interp, 0f, 0f, 1f);
 	}
 
+	private void renderWeaponSprintMovement(GunAnimations animations, ModelGun model, GunType gunType) {
+		Vector3f defaultTranslate = new Vector3f(0, 0F, -0.2);
+		Vector3f defaultRotation = new Vector3f(-15F, 45F, -10F);
+
+		Vector3f configuredTranslate = new Vector3f(model.sprintStanceTranslate.x, model.sprintStanceTranslate.y, model.sprintStanceTranslate.z);
```

Comment:
I'm not sure these deep copies are needed, unless I'm missing something

Reply:
I dont quite follow what you're referring to here

Reply:
As in, `Vector3f newthingy = new Vector3f(thingy.x, ....)` creates a new copy of the vector in the function, just for it to be read later on. 

if you want the name to be more readable in the function, just `Vector3f newthingy = thingy` should be fine. It gets more compliacted if you're modifying the value of newthingy, but that's not happenning here.

Reply:
Ah gotcha totally missed that lol